Components/BlockPostHeader/index.twig

```diff
@@ -0,0 +1,25 @@
+<div class="flyntComponent" is="flynt-block-post-header">
+  <div class="container">
+    <div class="blockPostHeader">
+      <div class="blockPostHeader-title">
+        <h1 class="blockPostHeader-h1">{{ post.title }}</h1>
+      </div>
+      <div class="blockPostHeader-meta">
+        {% if post.date %}
+          <time class="blockPostHeader-date">{{ post.date|date }}</time> {{ generalTranslatable.authorDivider|e }}
+        {% endif %}
+        {{ generalTranslatable.postedByLabel|e }} <a class="blockPostHeader-author" href="{{ post.author.link|e('esc_url') }}">{{ post.author.name|capitalize }}</a>
+        {{ generalTranslatable.postedInLabel|e }} <a class="blockPostHeader-category" href="{{ post.category.link|e('esc_url') }}">{{ post.category }}</a>
+        {{ generalTranslatable.readingtimeDivider|e }} <span class="blockPostHeader-readingtime">{{ post.content|readingtime }} {{ generalTranslatable.readingtimeLabel|e }}</span>
+      </div>
+      <div class="blockPostHeader-intro">
+        <p>{{ post.preview.read_more(false) }}</p>
```

Comment:
Does this work now also with not having a manual excerpt, or does it show the beginning of the actual post content instead?